stdlib/public/core/StringBridge.swift

```diff
@@ -306,6 +301,30 @@ public final class _NSContiguousString : _SwiftNativeNSString {
     return try body(selfAsPointer, rhsAsPointer)
   }
 
+  @objc
+  var fastestEncoding: UInt {
+    return _core.elementWidth == 1 ? UInt(1) : UInt(10)
```

Comment:
Would it make sense to sink this into an extension in the Foundation module, so we could use the real constants?